chart/tests/test_pgbouncer.py

```diff
@@ -427,7 +453,7 @@ def test_default_exporter_secret(self):
     def test_exporter_secret_with_overrides(self):
         connection = self._get_connection(
             {
-                "pgbouncer": {"enabled": True},
+                "pgbouncer": {"enabled": True, "metricsExporterSidecar": {"sslmode": "disable"}},
```

Comment:
```suggestion
                "pgbouncer": {"enabled": True, "metricsExporterSidecar": {"sslmode": "require"}},
```

The intent of this test is to test with non-default values. It also needs to change in the assert below, but I can't suggest it via the UI.